src/components/AnalysisPage/UnitsSelector/UnitsSelector.jsx

```diff
@@ -54,20 +54,28 @@ const UnitsSelector = ({ partId }) => {
     }
   };
 
+  const isDisabled = !partId;
+  const tooltipTitle = partId
+    ? ""
+    : "View or upload a part to the viewer before selecting a unit of measurement.";
+
```

Comment:
This tooltip is great! Is it possible to also make the unit selector less greyed out when there is a part in the page?

I feel like it looks pretty greyed out and unclickable even when their is a part.
![unit-selector-red](https://github.com/rknizzle/propermesh-view/assets/40612461/f3baceb9-65cc-40c4-8243-88c6c038b1c4)



Is that an easy change?

Reply:
If its not an easy change, then dont worry about this. Its not a big deal

Reply:
@rknizzle yeah it should be an easy change!! 

I'll check it out this evening

Reply:
Just changing the color of the words "select units" to black when they are able to select a measurement cleared that up If you ask me!

Reply:
Yup this looks awesome :+1: